### PR TITLE
2019-01-28 Renting a property

### PR DESCRIPTION
**Pull request by MerryODS**
…nships-support-better-rentals.md

Jac, can you try and figure out how to do the footnote properly (or we take it out or just reference it at the bottom)

### CODE REVIEW

**Comment by Br3nda** [comment]
This looks great! will review properly tomorrow

**Comment by MerryODS** [comment]
Hi Glen

Is that not what you wanted to do?  or just set up a page to link to the
posts?  It was more of a learning exercise to get me up to scratch for when
I am  2 weeks bedridden.

Let us know and we can take it from there.

In regards to the report - easier/cleaner/simpler is better... but up to you

let me know steps from here

M

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+With a short term discovery, it wasn’t possible to tackle all the themes and problems, so two were selected to investigate further by looking at the causes and coming up with ideas to mitigate them:
+
+- Security of tenure of a home for a period of time
+- Many landlords and property managers have a lack of experience around tenancy
+
+From here we identified several high level ideas to show what future interventions might be. Due to limited time, each of these ideas and the evidence that supports them, will need to be validated in more detail to ascertain the true value of progressing this work further.
+
+## What's next
+
```
Comment (JacOng17) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]: Extra empty line.

**Comment by JacOng17** [comment]
@MerryODS Well done! Your footnote works, just that you can't see from the preview but have attached screenshots of how it looks like.
Screenshot from 2019-09-30 09-47-01
Screenshot from 2019-09-30 09-46-44

**Comment by JacOng17** [comment]
Hi @gthurstonnz ,

In regards to your question: 
1. This is a copy of the blog post put on the digital.govt.nz. Are we copying our blog posts to the toolkit? **I'm not too sure, just approving the content but happy to take the lead from you.**

2. I have a monster report from MBIE that could be added but I was thinking of doing a cut down version of the report and adding this to the report section of property card. But looking at this blog post I am now thinking that we could just take the content from the blog post and expand it a little as its well written for what we need. I'm interested in your thoughts on this? **Sounds good, saves us needing to cut down huge reports** :+1:

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- 31% of rental houses feel damp, compared to 11% of owner- occupied houses
+- 22% of rental homes have no fixed heating sources
+- 56% of rental homes had visible mould, compared to 44% of owner-occupied houses[^1]
+
+The importance of rental accomodation led the Service Innovation governance group, which MBIE belongs to, to prioritise ‘Renting a Property’as a life event to explore from a holistic, all-of-government perspective.
+
+## How did we do it
+
+Our initial design statement was based on a hypothesis that a better rental house will result in a better relationship between tenant and landlord. The aim was to find out more about these relationships and the house itself. We essentially began with a fact finding mission as a foundation for future work.
```
Comment (JacOng17) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
Our initial design statement was based on a hypothesis that a better rental house will result in a better relationship between tenant and landlord. The aim was to find out more about these relationships and the house itself. We essentially began with a fact finding mission as a foundation for future work.
```

Comment (JacOng17) [reply _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]: Trailing whitespace at the end of line.

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## How did we do it
+
+Our initial design statement was based on a hypothesis that a better rental house will result in a better relationship between tenant and landlord. The aim was to find out more about these relationships and the house itself. We essentially began with a fact finding mission as a foundation for future work. 
+Significant research had already been undertaken. We built on this, reviewing over 48 documents that provided insights into renting from landlord, property manager and tenant perspectives and conducted in-depth interviews with 60 stakeholders. We also explored wellbeing frameworks relevant to renting, including the Māori concept of [Kāinga](https://www.digital.govt.nz/blog/kainga-and-renting-a-property/).
+
+## What did we learn
+
+We validated our original design statement by the end of week three, but with a twist. It was actually better relationships that influence the quality of the homes. Tenants who had good relationships were less likely to go into dispute with their landlords, more likely to raise issues, and see improvements, thus increasing house quality. As a result we revised our design statement:
```
Comment (JacOng17) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
We validated our original design statement by the end of week three, but with a twist. It was actually better relationships that influence the quality of the homes. Tenants who had good relationships were less likely to go into dispute with their landlords, more likely to raise issues, and see improvements, thus increasing house quality. As a result we revised our design statement:
```

Comment (JacOng17) [reply _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]: Trailing whitespace at the end of line.

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Significant research had already been undertaken. We built on this, reviewing over 48 documents that provided insights into renting from landlord, property manager and tenant perspectives and conducted in-depth interviews with 60 stakeholders. We also explored wellbeing frameworks relevant to renting, including the Māori concept of [Kāinga](https://www.digital.govt.nz/blog/kainga-and-renting-a-property/).
+
+## What did we learn
+
+We validated our original design statement by the end of week three, but with a twist. It was actually better relationships that influence the quality of the homes. Tenants who had good relationships were less likely to go into dispute with their landlords, more likely to raise issues, and see improvements, thus increasing house quality. As a result we revised our design statement: 
+>For tenants and landlords renting a property, create ways to raise recognition that better relationships between them aid in quality rental homes which will help lead to better outcomes for both.
+
+ Key themes and patterns of behaviour to explore further:
+
```
Comment (JacOng17) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
```

Comment (JacOng17) [reply _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]: Trailing whitespace at the end of line.

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+date: 2019-01-28
+authors:
+- glen-thurston
+tags:
+- engagement
+- collaboration
+- renting a property
+excerpt: ""
+---
+## Summary of the research report
```
Comment (Br3nda) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
## Summary of the research report
```

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+## What did we learn
+
+We validated our original design statement by the end of week three, but with a twist. It was actually better relationships that influence the quality of the homes. Tenants who had good relationships were less likely to go into dispute with their landlords, more likely to raise issues, and see improvements, thus increasing house quality. As a result we revised our design statement:
+>For tenants and landlords renting a property, create ways to raise recognition that better relationships between them aid in quality rental homes which will help lead to better outcomes for both.
+
+ Key themes and patterns of behaviour to explore further:
+1. **Relationships and communication** - Good relationships are essential to trust and transparency. This enables all parties to better understand each other, and make it more likely to facilitate ways to help each other. As a result, all parties had higher chances of securing tenure, trusting one another in the longer-term and improving rental stock.
```
Comment (Br3nda) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion

1. **Relationships and communication** - Good relationships are essential to trust and transparency. This enables all parties to better understand each other, and make it more likely to facilitate ways to help each other. As a result, all parties had higher chances of securing tenure, trusting one another in the longer-term and improving rental stock.
```

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+5. **The basics are essential to living** - Tenants, landlords and property managers expressed the importance of having the essentials (including social and community networks) close by rentals to meet basic daily needs of tenants. Many tenants were either unable to access or were unaware of tenancy support services.
+
+With a short term discovery, it wasn’t possible to tackle all the themes and problems, so two were selected to investigate further by looking at the causes and coming up with ideas to mitigate them:
+
+- Security of tenure of a home for a period of time
+- Many landlords and property managers have a lack of experience around tenancy
+
+From here we identified several high level ideas to show what future interventions might be. Due to limited time, each of these ideas and the evidence that supports them, will need to be validated in more detail to ascertain the true value of progressing this work further.
+
+## What has the research been used for?
```
Comment (Br3nda) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
## Using our research
```

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+- Security of tenure of a home for a period of time
+- Many landlords and property managers have a lack of experience around tenancy
+
+From here we identified several high level ideas to show what future interventions might be. Due to limited time, each of these ideas and the evidence that supports them, will need to be validated in more detail to ascertain the true value of progressing this work further.
+
+## Using our research
+
+Our partners, MBIE Tenancy Services, have used this work for: 
+- Design and development of a possible Tenancy Agreement Builder – direct outcome/concept from the discovery, supported by the evidence gathered.
```
Comment (Br3nda) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
- Design and development of a possible Tenancy Agreement Builder – direct outcome/concept from the discovery, supported by the evidence gathered.
```

File: _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+With a short term discovery, it wasn’t possible to tackle all the themes and problems, so two were selected to investigate further by looking at the causes and coming up with ideas to mitigate them:
+
+- Security of tenure of a home for a period of time
+- Many landlords and property managers have a lack of experience around tenancy
+
+From here we identified several high level ideas to show what future interventions might be. Due to limited time, each of these ideas and the evidence that supports them, will need to be validated in more detail to ascertain the true value of progressing this work further.
+
+## Using our research
+
+Our partners, MBIE Tenancy Services, have used this work for:
```
Comment (Br3nda) [inline _posts/2019-01-28-renting-a-property-understanding-how-better-relationships-support-better-rentals.md]:
```suggestion
Our partners, MBIE Tenancy Services, have used this work for:

```

**Comment by Br3nda** [comment]
I have rebased and squashed this branch.

**Pull request merged**